Wrapping/Generators/SwigInterface/CMakeLists.txt

```diff
@@ -393,7 +393,6 @@ macro(itk_end_wrap_module_swig_interface)
         )
       else()
         unset(ITK_STUB_DIR)
-        unset(ITK_STUB_PYI_FILE)
```

Comment:
NOTE: No 'S; at the end of ITK_STUB_PYI_FILE.  The variable name should have been ITK_STUB_PYI_FILES.